Gaming The Syllabus/Assets/Scripts/Combat/Action Effects/ActionEffect.cs

```diff
@@ -17,10 +17,10 @@ public abstract class ActionEffect : IXmlSerializable
     If AbilityCaller is null, this means that this action effect is not being called
     by a PlayerAbility.
     */
-    public PlayerAbility AbilityCaller {get {return _AbilityCaller;}
-                                        set { if (_AbilityCaller == null)
-                                                    _AbilityCaller = value;
-                                                else return;}}
+    // public PlayerAbility AbilityCaller {get {return _AbilityCaller;}
```

Comment:
We're eliminating AbilityCaller completely right? If so might as well go ahead and delete this commented block if you're comfortable with it!